src/components/UserManagement/UserManagement.test.tsx

```diff
@@ -0,0 +1,33 @@
+import { render, screen } from '@testing-library/react';
+import UserManagement from './UserManagement';
+import userEvent from '@testing-library/user-event';
+
+describe('UserManagement', () => {
+  it('renders the UserManagement component', () => {
+    render(<UserManagement />);
+    const userManagementElement = screen.getByTestId('user-management');
```

Comment:
I would say if you need to use it in multiple locations it makes sense to put in variable. But may be cleaner if we just use inline?